doc/source/security.md

```diff
@@ -155,3 +155,63 @@ This is a sensitive security issue (similar to writing sudo rules in a
 traditional computing environment), so be very careful.
 
 There's ongoing work on making this easier!
+
+## Advanced: Adding your own ingress object
```

Comment:
I think this should still be in advanced and not security, since Ingress is not about security at all :)